cmd/admin-handlers.go

```diff
@@ -312,6 +314,31 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	writeSuccessResponseJSON(w, jsonBytes)
 }
 
+// ServerInfoHandler - GET /minio/admin/v1/storageinfo
+// ----------
+// Get server information
+func (a adminAPIHandlers) StorageInfoHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think before these changes we need to fix the API versioning issue with admin-handlers, such that we can bring these new changes.

Reply:
yes